src/api/transaction/utils.js

```diff
@@ -54,11 +58,13 @@ function prepareTransaction(transaction: any, remote: any, instructions: any,
 
   function prepareMaxLedgerVersion(callback_) {
     if (instructions.maxLedgerVersion !== undefined) {
-      txJSON.LastLedgerSequence = parseInt(instructions.maxLedgerVersion, 10);
+      txJSON.LastLedgerSequence =
+        typeof instructions.maxLedgerVersion === undefined ?
```

Comment:
I added this to preserve old behavior - `parseInt` returns `NaN` if value isn't something like digit.
If it is not needed, I can just do assign here.